File: ClientComponents/BarkFluff.WebApi.Core/MessengerData/NonSavedData/ServerDataElement.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BarkFluff.WebApi.Core.MessengerData.NonSavedData
{
    public class ServerDataElement
    {
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Ip { get; set; } = string.Empty;
        public string UserCount {  get; set; } = string.Empty;

    }
}

File: Android/core/src/main/java/com/barkfluff/client/data/ServerDataElement.kt
package com.barkfluff.client.data

/**
 * Данные о сервере для отображения в списке
 */
data class ServerDataElement(
    val title: String = "",
    val description: String = "",
    val ip: String = "",
    val userCount: String = "",
    val publicName: String = "",
    val location: String = "",
    val hexColor: String = "",
    val filesMediaEndpoint: String = ""
)
